ugvc/mrd/bqsr_train_utils.py

```diff
@@ -0,0 +1,807 @@
+from __future__ import annotations
+
+import json
+import os
+import random
+import subprocess
+from os.path import isfile
+from os.path import join as pjoin
+
+import numpy as np
+import pandas as pd
+import xgboost as xgb
+import pysam
+from tqdm import tqdm
+from collections import defaultdict
+
+from joblib import dump
+from matplotlib import pyplot as plt
+from sklearn.model_selection import train_test_split
+
+from ugvc.dna.format import CHROM_DTYPE, DEFAULT_FLOW_ORDER
+from ugvc.dna.utils import get_max_softclip_len, revcomp
+from ugvc.utils.consts import FileExtension
+from ugvc.vcfbed.variant_annotation import get_cycle_skip_dataframe, get_motif_around
+from ugvc.dna.format import DEFAULT_FLOW_ORDER
+from ugvc.mrd.mrd_utils import featuremap_to_dataframe
+from ugvc.utils.misc_utils import set_pyplot_defaults
+
+set_pyplot_defaults()
+
+
+def model_params(params):
+
+    params["balance_motifs"] = True
+    params["eval_only"] = False
```

Comment:
This is not used. One of the reasons that I think this dictionary structure is not good is that the interpreter can't figure that out this way.